steam/webauth.py

```diff
@@ -224,6 +224,8 @@ def login(self, password='', captcha='', email_code='', twofactor_code='', langu
                 raise EmailCodeRequired(resp['message'])
             elif resp.get('requires_twofactor', False):
                 raise TwoFactorCodeRequired(resp['message'])
+            elif 'short time period' in resp.get('message', ''):
```

Comment:
Best to search for `too many login failures` instead here

Reply:
> Best to search for `too many login failures` instead here

okay !